strings/src/main/java/com/revenuecat/purchases/strings/AttributionStrings.kt

```diff
@@ -1,5 +1,36 @@
 package com.revenuecat.purchases.strings
 
 object AttributionStrings {
-    const val NO_SUBSCRIBER_ATTRIBUTES_TO_SYNCHRONIZE = "No subscriber attributes to synchronize."
+    const val ATTRIBUTE_ERROR = Emojis.SAD_CAT_EYES + Emojis.ERROR + " There were some subscriber attributes errors: %s"
+    const val ATTRIBUTES_SYNC_SUCCESS = Emojis.HEART_CAT_EYES + " Subscriber attributes synced successfully for " +
+            "App User ID: %s"
+    const val COLLECT_DEVICE_IDS = Emojis.INFO + " collectDeviceIdentifiers called"
+    const val DELETE_ATTRIBUTES = Emojis.INFO + " Deleting subscriber attributes for %s from cache"
```

Comment:
Aside from the comment about emojis in a method, forcing devs to start each of these with a space `" "` seems error-prone. We could probably have these be a list that's joined by the `" "` character

Reply:
something along the lines of
```kotlin
const val DELETE_ATTRIBUTES = [Emojis.INFO, " Deleting subscriber attributes for %s from cache"].joinToString(separator = " ")
```

Reply:
since we'd use that method often, we could create a convenience for it so that we don't have to specify the separator every time, but that'd be the spirit of it